tutorials/01_introductory/usage.py

```diff
@@ -625,3 +625,95 @@ def my_plotter(ax, data1, data2, param_dict):
 # or generating a new set of figures.  In that case, use
 # :func:`~matplotlib.pyplot.show` to display the figure(s) and
 # to block execution until you have manually destroyed them.
+#
+# .. _performance:
+#
+# Performance
+# ===========
+#
+# Whether exploring data in interactive mode or programatically
+# saving lots of plots, rendering performance can be a painful
+# bottleneck in your pipeline. Matplotlib provides a couple
+# ways to greatly reduce rendering time at the cost of a slight
+# change (to a settable tolerance) in your plot's appearance.
+# The methods available to reduce rendering time depend on the
+# type of plot that is being created.
+#
+# Line segment simplification
+# ---------------------------
+#
+# For plots that have line segments (e.g. typical line plots,
+# outlines of polygons, etc.), rendering performance can be
+# controlled by the ``path.simplify`` and
+# ``path.simplify_threshold`` parameters in your
+# ``matplotlibrc`` file (see
+# :ref:`sphx_glr_tutorials_01_introductory_customizing.py` for
+# more information about the ``matplotlibrc`` file).
+# The ``path.simplify`` parameter is a boolean indicating whether
+# or not line segments are simplified at all. The
+# ``path.simplify_threshold`` parameter controls how much line
+# segments are simplified; higher thresholds result in quicker
+# rendering.
+#
+# The following script will first display the data without any
+# simplification, and then display the same data with simplification.
+# Try interacting with both of them::
+#
+#   import numpy as np
+#   import matplotlib.pyplot as plt
+#   import matplotlib as mpl
+#
+#   # Setup, and create the data to plot
+#   y = np.random.rand(100000)
+#   y[50000:] *= 2
+#   y[np.logspace(1,np.log10(50000),400).astype(int)] = -1
+#   mpl.rcParams['path.simplify'] = True
+#
+#   mpl.rcParams['path.simplify_threshold'] = 0.0
+#   plt.plot(y)
+#   plt.show()
+#
+#   mpl.rcParams['path.simplify_threshold'] = 1.0
+#   plt.plot(y)
+#   plt.show()
+#
+# Matplotlib currently defaults to a conservative simplification
+# threshold of ``1/9``. If you want to change your default settings
+# to use a different value, you can change your ``matplotlibrc``
+# file.  Alternatively, you could create a new style for
+# interactive plotting (with maximal simplification) and another
+# style for publication quality plotting (with minimal
+# simplification) and activate them as necessary. See
+# :ref:`sphx_glr_tutorials_01_introductory_customizing.py` for
```

Comment:
Note: Current version of that PR would change it to `sphx_glr_tutorials_introductory_customizing.py`